include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -208,6 +210,53 @@ namespace parallel
     };
 
 
+
+    /**
+     * Struct used in
+     * parallel::fullydistributed::Triangulation::copy_triangulation() to create
+     * the ConstructionData for a user provided Triangulation. It can be
+     * registered to a parallel::fullydistributed::Triangulation via the
+     * function
+     * parallel::fullydistributed::Triangulation::register_dynamic_construction_data().
+     */
+    template <int dim, int spacedim = dim>
+    struct DynamicConstructionData
+    {
+      /**
+       * Constructor.
+       *
+       * @note As a default, GridTools::partition_triangulation_zorder() is used
+       *       as partitioner and data structures on multigrid levels are not
+       * set up.
+       */
+      DynamicConstructionData(
+        std::function<void(dealii::Triangulation<dim, spacedim> &,
+                           const unsigned int)> partitioner =
+          [](dealii::Triangulation<dim> &tria,
+             const unsigned int          n_partitions) {
```

Comment:
It would be useful if you could document what the arguments of the function argument are.